src/app/forms/components/date/date.component.ts

```diff
@@ -50,6 +50,7 @@ export class DateComponent extends BaseControlComponent implements OnInit, OnDes
   public year?: number;
   public hour?: number;
   public minute?: number;
+  showError: boolean = false;
```

Comment:
Is this meant to be public, or private? It'd be better to be specific about it like in the other components.